spec/features/members/member_deletion_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+
+feature "member deletion" do
+
+  context "with activity and followers" do
+    let(:member) { FactoryGirl.create(:member) }
+    let(:other_member) { FactoryGirl.create(:member) }
+    let(:memberpost) { FactoryGirl.create(:post, :author => member) }
+    let(:othermemberpost) { FactoryGirl.create(:post, :author => other_member) }
+    let(:planting) { FactoryGirl.create(:planting, :owner => member) }
+    let(:harvest) { FactoryGirl.create(:harvest, :owner => member) }
+    let(:seed) { FactoryGirl.create(:seed, :owner => member) }
+    let(:secondgarden) { FactoryGirl.create(:garden, :owner => member) }
+    background do
+      login_as(member)
+      visit member_path(other_member)
+      click_link 'Follow'
+      logout
+      login_as(other_member)
+      visit member_path(member)
+      click_link 'Follow'
+      logout
+      login_as(member)
+      FactoryGirl.create(:comment, :author => member, :post => othermemberpost)
+      FactoryGirl.create(:comment, :author => other_member, :post => memberpost)
+    end
+
+    scenario "has option to delete on member profile page" do
+      visit member_path(member)
+      expect(page).to have_link "Delete account"
+    end
+    
+    scenario "requests confirmation for deletion"
+    # need to figure out how to test javascript if we want to bother
```

Comment:
Just be sure to use :js => true to make turn on Phantom.

iPhone$B$+$iAw?.(B

2015/01/21 2:07$B!"(Bpozorvlak <notifications@github.com> $B$N%a%C%;!<%8(B:

> In spec/features/members/member_deletion_spec.rb:
> 
> > +      login_as(other_member)
> > +      visit member_path(member)
> > +      click_link 'Follow'
> > +      logout
> > +      login_as(member)
> > +      FactoryGirl.create(:comment, :author => member, :post => othermemberpost)
> > +      FactoryGirl.create(:comment, :author => other_member, :post => memberpost)
> > +    end
> > +
> > +    scenario "has option to delete on member profile page" do
> > +      visit member_path(member)
> > +      expect(page).to have_link "Delete account"
> > +    end
> > +    
> > +    scenario "requests confirmation for deletion"
> > +    # need to figure out how to test javascript if we want to bother
> Testing JavaScript is no longer a problem - PhantomJS (which we're using for feature tests) embeds a full JavaScript interpreter.
> 
> $B!=(B
> Reply to this email directly or view it on GitHub.
>